src/components/Branding/styled-components.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingContainer = styled.div`
+  position: fixed;
+  top: 0px;
+  width: inherit;
+  overflow-wrap: break-word;
+  text-align: center;
+  font-size: 28px;
+  letter-spacing: -2px;
+  margin-top: 8px;
```

Comment:
Our standard spacing is 16px. I don't mind having it smaller since text have empty top and bottom spacing. But 8px seems too low.

Reply:
Changed margin-top to 16px. Looks uniform.